collectors/ebpf.plugin/README.md

```diff
@@ -116,8 +192,33 @@ processes](https://en.wikipedia.org/wiki/Zombie_process).
 
 #### Task error
 
-The functions responsible for ending tasks do not return values, so this chart contains information about failures on
-process and thread creation.
+The functions responsible for ending tasks do not return values, so this chart contains information about failures on 
+process and thread creation only.
+
+#### TCP functions
+
+This chart demonstrates calls to functions `tcp_sendmsg`, `tcp_cleanup_rbuf` and `tcp_close`; these functions are
+used to send & receive data and to close connections when `TCP` protocol is used.
+
+#### TCP bandwidth
+
+Like the previous section, this chart also monitors `tcp_sendmsg` and `tcp_cleanup_rbuf` but, instead of showing the 
```

Comment:
```suggestion
Like the previous chart, this one also monitors `tcp_sendmsg` and `tcp_cleanup_rbuf` but, instead of showing the 
```
BTW, why there is the `close` dimension in this chart? Shouldn't it be deleted?

Reply:
Also comma here should be before not after "but"

Reply:
@vlvkobal I believe @underhood is right.

Reply:
@vlvkobal you are right, I will remove the dimension in another PR.

@underhood and @UmanShahzad  I had a long talk with @hugovalente-pm about this, and I am glad I am not the unique with this opinion, I am moving the comma.